cron/monthly/invoice-platform-fees.js

```diff
@@ -384,6 +385,7 @@ export async function run() {
           netAmountInCollectiveCurrency: totalAmountCredited,
           type: TransactionTypes.CREDIT,
           TransactionGroup: uuid(),
+          kind: TransactionKind.PLATFORM_TIP,
```

Comment:
I would keep it `null` as this should normally be deprecated in the process of our refactoring.

This will be replaced by having `kind=PLATFORM_TIP` `isDebt=true` transactions one by one. Instead of batching them at the end of the month.